From: Armin Burgmeier <armin@arbur.net>
Date: Fri, 22 May 2009 18:25:55 +0000 (+0200)
Subject: Locate gnutls via pkg-config, not libgnutls-config
X-Git-Url: http://git.0x539.de/?p=net6.git;a=commitdiff_plain;h=1a1216523ef29c9f7168f0d2e1917edd4463a142

Locate gnutls via pkg-config, not libgnutls-config

2009-05-22  Armin Burgmeier  <armin@0x539.de>

	* configure.ac: Locate gnutls via pkg-config, not libgnutls-config
	which newer versions no longer have.
---

diff --git a/ChangeLog b/ChangeLog
index 76875c1..a2cfd00 100644
--- a/ChangeLog
+++ b/ChangeLog
@@ -1,3 +1,8 @@
+2009-05-22  Armin Burgmeier  <armin@0x539.de>
+
+	* configure.ac: Locate gnutls via pkg-config, not libgnutls-config
+	which newer versions no longer have.
+
 2009-01-03  Philipp Kern  <phil@0x539.de>
 
 	* configure.ac:
diff --git a/configure.ac b/configure.ac
index b246cc5..401083c 100644
--- a/configure.ac
+++ b/configure.ac
@@ -72,18 +72,7 @@ AC_SUBST(addlibs)
 AM_CONDITIONAL(WIN32, test x$win32 = xtrue)
 
 # Checks for libraries.
-PKG_CHECK_MODULES([libnet6], [sigc++-2.0])
-
-AM_PATH_LIBGNUTLS( 1.0.0,, AC_MSG_ERROR([[
-***
-*** libgnutls was not found. You may want to get it by using the
-*** package manager of your distribution or by compiling the source
-*** from ftp://ftp.gnutls.org/pub/gnutls/. This dependency is
-*** now mandatory.
-]]))
-addlibs="$addlibs $LIBGNUTLS_LIBS"
-addcflags="$LIBGNUTLS_CFLAGS"
-AC_SUBST(addcflags)
+PKG_CHECK_MODULES([libnet6], [sigc++-2.0, gnutls])
 
 # gettext / i18n
 AM_GNU_GETTEXT([external])
